### PR TITLE
Mergeup 5.3.x to master

### PR DESCRIPTION
**Pull request by ScottGarman**

### CODE REVIEW

File: configs/components/cpp-pcp-client.json [hunk 1]
```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/cpp-pcp-client.git", "ref": "fbde5d06f846cb649a5f7a296da0e83ef9d86fb4"}
```
Comment (MikaelSmith) [inline configs/components/cpp-pcp-client.json]: Do we want the tagged version here?

**Comment by MikaelSmith** [comment]
There's more code to merge-up, if puppet-agent#5.3.x succeeds tonight we should probably repeat the merge.

**Comment by ScottGarman** [comment]
Yeah, if this gets merged it's just going to break cross-compiled builds on master. I'll generate a new mergeup tomorrow morning after 5.3.x succeeds.

File: configs/components/cpp-pcp-client.json (hunk 1 above)
Comment (MikaelSmith) [reply configs/components/cpp-pcp-client.json]: The merge instructions do `for i in {hiera,facter,puppet,marionette-collective,pxp-agent,cpp-pcp-client}; do git checkout master -- configs/components/$i.json;done`, which specifically excludes changes to those json files.

Comment (ScottGarman) [reply configs/components/cpp-pcp-client.json]: Ok, I'll make a note of that for future reference and leave that out of my re-doing of this merge-up PR in a bit.

**Comment by ScottGarman** [comment]
Closing this in favor of: https://github.com/puppetlabs/puppet-agent/pull/1178

**Pull request closed without merging**